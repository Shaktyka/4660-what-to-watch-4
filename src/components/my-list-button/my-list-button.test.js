import React from 'react';

import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import MyListButton from './my-list-button.jsx';

describe(`MyListButton rendering`, () => {

  it(`MyListButton renders correctly`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <MyListButton
              id={1}
              isFavorite={true}
              onClick={() => {}}
            />
          </BrowserRouter>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
