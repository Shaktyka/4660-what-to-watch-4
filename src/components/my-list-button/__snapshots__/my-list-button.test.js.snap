// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyListButton rendering MyListButton renders correctly 1`] = `
<a
  className="btn btn--list movie-card__button"
  href="/mylist"
  onClick={[Function]}
>
  <svg
    height="14"
    viewBox="0 0 18 14"
    width="18"
  >
    <use
      xlinkHref="#in-list"
    />
  </svg>
  <span>
    My list
  </span>
</a>
`;
